simpl/games/models.py

```diff
@@ -38,8 +38,7 @@ class Decision(AbstractTimeStampedModel):
     )
 
     class Meta(object):
-        # Instructor decisions are not associated with a role, so cannot:
-        # unique_together = ('name', 'period', 'role')
+        unique_together = ('name', 'period', 'role')
```

Comment:
Are we sure that having null roles here doesn't cause problems?

Reply:
I tested having null roles using simpl-calc-ui. It's also the same constraint that we already have on Result.